examples/demo-react-native/detox.config.js

```diff
@@ -51,6 +51,14 @@ module.exports = {
       "device": {
         "avdName": "Pixel_API_28"
       }
+    },
+    "android.att.release": {
```

Comment:
Perhaps change to `android.genymotion.release`? a bit confusing to see both `attached` and the `emulator-.*` regexp in the same config

Reply:
I asked @DmytroKost first of all to add it for the testing purposes. I am sure that Genymotion emulators do not follow `emulator-<port>` naming (!). But we can add something Genymotionish as well.

But maybe it's indeed good to rename it to `android.any.release` (att -> any).

Reply:
We connect to Genymotion via SSH tunnel, the names of devices will be `localhost:5555`, `localhost:5556`, .etc.

I can make an example for `android.genymotion.release` to demonstrate that, and one for `android.any.release` with a `.*` mark to demonstrate connection to any attached device/emulator. I thought `detox.config.js` for also for demo purposes. But since it's for testing, I'm not sure I should make any changes to it.

I also mentioned the example with a remote device in https://github.com/DmytroKost/Detox/blob/uic/adb-name-array/docs/APIRef.Configuration.md#device-configuration

Reply:
@d4vidi, your call.

Reply:
@d4vidi , @DmytroKost , let's change it to ".*" and rename to `any` instead of `att`. It is just for local testing at the moment.

Reply:
Any progress here? @d4vidi, is the `.*` suggestion okay?

Reply:
Why `.*`? does it really make sense to match all-the-things? How about we negate `emulator-`? `(?!(emulator))`, I believe...

Anyways on 2nd thought no point in adding an extra genymotion config here - I agree.

Reply:
Or iso negating, assert that it's a remotely connected device with [this pattern](https://github.com/DmytroKost/Detox/blob/uic/adb-name-array/detox/src/devices/drivers/android/tools/DeviceHandle.js#L14)

Reply:
@d4vidi @noomorph Reverted changes to the `detox.config.js` file. Moved the example with `android.genymotion.release` to `docs/APIRef.Configuration.md`. What do you think?